setup.cfg

```diff
@@ -98,7 +98,6 @@ norecursedirs =
     dist
     build
 addopts =
-    --strict
```

Comment:
Removed because it is deprecated -- `The --strict option is deprecated, use --strict-markers instead.`